### ISSUE TITLE
router-link lost class attr when render to <a>

### ISSUE THREAD

**Issue by shellvip** [KingPixil/moon-router#2]
```
<router-link class="add-picture-button mdl-button mdl-js-button mdl-button--fab" to="/post">
  <i class="material-icons">add</i>
</router-link>
```
render to:
```
<a href="#/post">
  <i class="material-icons m-scope-ykj5am">add</i>
</a>
```
lost the class attr.

**Comment by kbrsh** [comment]
Thanks for the report, this will be fixed in Moon Router v0.0.6 and Moon v0.11.0.

**Comment by findborg** [comment]
I second this,  we definitely need to be able to assign classes to the links.
